### ISSUE TITLE
Player does not play video on iOS with "filePath not implemented" message

### ISSUE THREAD

**Issue by nirajhinge** [jepiqueau/capacitor-video-player#61]
I am trying to use the player to play fullscreen videos on iOS and Android. I am currently testing it on iOS with the following code:

```js
 const playVideo = async () => {
      const url = props.attachment.path
        ? props.attachment.path
        : props.attachment.temporary_url
      const init = await CapacitorVideoPlayer.initPlayer({
        mode: 'fullscreen',
        url: url,
        playerId: 'fullscreen',
      })
      console.log('const init ', init)
      const play = await CapacitorVideoPlayer.play({ playerId: 'fullscreen' })
      console.log('const play ', play)
    }
```
When the video is loaded from a web it has a `.mp4` format like this: `https:://www.foo.bar/video.mp4`. This works as expected. I then have a button in the app where the user can select a video from `Photos` in iOS. After the video is selected, the path looks like this: `file:///var/mobile/Containers/Data/Application/EF0BB0B9-2DFE-4E56-B0DB-84683B777960/tmp/cdv_photo_1636993772.MOV` but then I get the error as you can see in the screenshot below:

![Screenshot 2021-11-15 at 16 37 13](https://user-images.githubusercontent.com/54309996/141819417-6c4f4bdf-9728-4413-8320-e6fc34af0d59.png)

Now based on what I have [read in the API](https://github.com/jepiqueau/capacitor-video-player/blob/master/docs/API.md#from-internal-folder), this might not be supported in iOS and it looks like a file path issue since if I try to use the same video from a https link, it works as expected. 

Is there a method where we can use the video player with videos uploaded locally on a path like this `file:///var/mobile/Containers/Data/Application/EF0BB0B9-2DFE-4E56-B0DB-84683B777960/tmp/cdv_photo_1636993772.MOV`? 

Thanks.

**Comment by jepiqueau** [comment]
@nirajhinge You have to give me a bit more of your code. what are the value of `props.attachment.path` and `props.attachment.temporary_url` . when you said ` I then have a button in the app where the user can select a video from Photos in iOS` when clicked how do you select the video from Photos. which plugin are you using to revert that path?. In `capacitor-video-player` if you use the `url= "internal"` you will be directed to the Media folder of your device and you can then select whatever video and play it fullscreen the path to the video folder is automatically created and be something like this `'file:///var/mobile/Containers/Data/PluginKitPlugin/FF27BEE4-C3F5-405A-926C-84FEA1D3583F/tmp/YOUR_SELECTED_VIDEO.MOV'` . This is due to the fact that i use the standard `UIImagePickerController` which when you accept `YOUR_SELECTED_VIDEO` it is first compressed.

When you have a path like this 
```
file:///var/mobile/Containers/Data/Application/EF0BB0B9-2DFE-4E56-B0DB-84683B777960/tmp/cdv_photo_1636993772.MOV
```
the application id `EF0BB0B9-2DFE-4E56-B0DB-84683B777960` is changed each time you build your app so the only way to implement what you want is in  the plugin to recreate the path given by 
`file:///var/mobile/Containers/Data/Application/`+ determine the current app id + `/tmp/cdv_photo_1636993772.MOV`

Is this what you want?

**Comment by jepiqueau** [comment]
@nirajhinge this has been fixed in 3.3.1-2, revert to me when tested

**Comment by nirajhinge** [comment]
@jepiqueau Sorry for the late reply but you are right in your assumptions. The video is selected using the [cordova native camera plugin](https://ionicframework.com/docs/native/camera) since for some reason I couldn't use capacitor camera plugin to select videos. A sample of the code looks like this:
```js
const cameraOptions: CameraOptions = {
      destinationType: this.camera.DestinationType.NATIVE_URI,
      sourceType: this.camera.PictureSourceType.SAVEDPHOTOALBUM,
      mediaType: this.camera.MediaType.VIDEO,
    };
    ````
All I needed was the path (temporary or permanent) for the compressed video to so that I could display an instant preview while the video gets uploaded so what you suggested worked even though I have decided to go for now with HTML5 video player (converting the .mov to .mp4). But if anyone wanted to use this plugin, that should work so thank you for your help.
